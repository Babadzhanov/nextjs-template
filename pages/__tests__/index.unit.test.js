import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';

import Index from '../index';

describe('<Index />', () => {
  it('renders correctly', () => {
    const index = shallow(<Index />);
    expect(toJson(index)).toMatchSnapshot();
  });
});
