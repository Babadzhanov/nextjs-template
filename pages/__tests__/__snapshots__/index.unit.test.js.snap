// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Index /> renders correctly 1`] = `
Array [
  <Meta
    description="A comprehensive Nextjs template"
    key="0"
    title="Batman Shows | Nextjs Template"
  />,
  <Layout
    key="1"
  />,
]
`;
